operators/operator_sdk/osdk-upgrading-projects.adoc

```diff
@@ -0,0 +1,25 @@
+[id="osdk-upgrading-projects"]
+= Upgrading projects for newer Operator SDK versions
+include::modules/common-attributes.adoc[]
+:context: osdk-upgrading-projects
+
+:osdk_ver: v1.8.0
+:osdk_ver_n1: v1.3.0
+:product-version-n1: 4.7
+
+toc::[]
+
+{product-title} {product-title} supports Operator SDK {osdk_ver}. If you already have the {osdk_ver_n1} CLI installed on your workstation, you can upgrade the CLI to {osdk_ver} by xref:../../operators/operator_sdk/osdk-installing-cli.adoc#osdk-installing-cli[installing the latest version].
+
+However, to ensure your existing Operator projects maintain compatibility with Operator SDK {osdk_ver}, upgrade steps are required for the associated breaking changes introduced since {osdk_ver_n1}. You must perform the upgrade steps manually in any of your Operator projects that were previously created or maintained with {osdk_ver_n1}.
+
+include::modules/osdk-upgrading-v130-to-v180.adoc[leveloffset=+1]
+
+[id="osdk-upgrading-projects-addtl-resources"]
```

Comment:
Civilization won't crumble if you prefer to keep this ID, but Paul did create a guideline:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies

Reply:
Ah so there is now. I checked the repo before choosing and saw both in use (probably mostly from me for the `addtl`), but I'll sync it up with guidelines.